ci/nightly/pipeline.yml

```diff
@@ -8,6 +8,13 @@
 # by the Apache License, Version 2.0.
 
 steps:
+  - id: build-x86_64
+    label: Build x86_64
+    command: bin/ci-builder run stable bin/pyactivate --dev -m ci.test.build x86_64
+    timeout_in_minutes: 60
+    agents:
+      queue: builder
```

Comment:
Would it be better to put this after the nightly test selection but before the trim pipeline? I.e., swap it with the next step. That way you don't have to wait for the build to complete before you get to select what tests to run.

Reply:
Done, thanks.